gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1878,32 +1878,22 @@ def test_{{ method_name }}_empty_call():
 {% endmacro %}
 
 
-{% macro transport_kind_test(service, opts, is_async=False) %}
-@pytest.mark.parametrize("transport_name", [
-    {% if is_async %}
-    {% if "grpc" in opts.transport %}
-    "grpc_asyncio",
-    {% endif %}
-    {% else %}{# if not is_async #}
-    {% if "grpc" in opts.transport%}
-    "grpc",
-    {% endif %}
-    {% if "rest" in opts.transport %}
-    "rest",
-    {% endif %}
-    {% endif %}{# is_async #}
-])
-{% if is_async %}
-@pytest.mark.asyncio
-async def test_transport_kind_async(transport_name):
-    transport = {{ service.async_client_name }}.get_transport_class(transport_name)(
-        credentials=async_anonymous_credentials(),
-    )
-{% else %}
-def test_transport_kind(transport_name):
-    transport = {{ service.client_name }}.get_transport_class(transport_name)(
-        credentials=ga_credentials.AnonymousCredentials(),
+{% macro transport_kind_test(service, transport_name) %}
+{%- set named_clients = {
```

Comment:
```suggestion
{% macro transport_kind_test(service, transport_name) %}
{# Look-up tables for parameter values corresponding to each possible transport-name #}
{%- set named_clients = {
```

Reply:
I ended up removing this dict since there's only two possible clients that we can have i.e. sync or async. Added a separate macro for it.

Reply:
Looks good!